src/Rings/MPolyQuo.jl

```diff
@@ -713,6 +721,26 @@ function simplify(f::MPolyQuoRingElem)
   return f::typeof(f)
 end
 
+# Extra method for quotients of graded rings. 
+# TODO: Could this be simplified if the type-parameter signature of decorated rings 
+# was consistent with the one for polynomial rings? I.e. if the first type parameter 
+# was the one for the coefficient rings and not the one for the underlying polynomial ring?
+function simplify(f::MPolyQuoRingElem{MPolyDecRingElem{T}}) where {T<:MPolyElem{<:FieldElem}}
```

Comment:
Probably, I should then also add the new functionality with `f.simplified` here, right? @thofma